docs/resources/dms_rocketmq_instance.md

```diff
@@ -0,0 +1,178 @@
+  The description can contain a maximum of 1024 characters.
+
+* `ssl_enable` - (Optional, Bool, ForceNew) Whether to enable SSL encryption for access.
+  Changing this parameter will create a new resource.
+
+* `ipv6_enable` - (Optional, Bool, ForceNew) Whether to support IPv6.
+  Changing this parameter will create a new resource.
+
+* `enable_publicip` - (Optional, Bool, ForceNew) Whether to enable public access.
+  By default, public access is disabled.
+  Changing this parameter will create a new resource.
+
+* `publicip_id` - (Optional, String, ForceNew) ID of the EIP bound to the instance.
+  Use commas (,) to separate multiple EIP IDs.
+  This parameter is mandatory if public access is enabled (that is, enable_publicip is set to true).
+  Changing this parameter will create a new resource.
+
+* `broker_num` - (Optional, Int, ForceNew) Number of brokers.
+  Changing this parameter will create a new resource.
+
+* `retention_policy` - (Optional, Bool) ACL.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `status` - MStatus.
+
+* `type` - Instance type. Value: cluster.
+
+* `specification` - Instance specification.
+
+* `instance_id` - Instance ID.
+
+* `charging_mode` - Billing mode. 1 indicates pay-per-use.
+
+* `vpc_name` - VPC name.
+
+* `security_group_name` - Security group name.
+
+* `subnet_name` - Subnet name.
+
+* `subnet_cidr` - Subnet route.
+
+* `user_id` - User ID.
+
+* `user_name` - Username.
+
+* `maintain_begin` - Time at which the maintenance window starts. The format is HH:mm:ss.
+
+* `maintain_end` - Time at which the maintenance window ends. The format is HH:mm:ss.
+
+* `enable_log_collection` - Whether log collection is enabled.
+
+* `used_storage_space` - Used message storage space. Unit: GB.
+
+* `publicip_address` - Public IP address.
+
+* `cross_vpc_info` - Cross-VPC access information.
+
+* `storage_resource_id` - Storage resource ID.
+
+* `service_type` - Service type.
+
+* `storage_type` - Storage category.
+
+* `extend_times` - Extension time.
+
+* `support_features` - Features supported by the instance.
+
+* `disk_encrypted` - Whether disk encryption is enabled.
+
+* `ces_version` - Cloud Eye version.
+
+* `node_num` - Node quantity.
+
+* `new_spec_billing_enable` - Whether billing based on new specifications is enabled.
+
+* `enable_acl` - Whether access control is enabled.
+
+* `namesrv_address` - Metadata address.
+
+* `broker_address` - Service data address.
+
+* `public_namesrv_address` - Public network metadata address.
+
+* `public_broker_address` - Public network service data address.
+
+* `total_storage_space` - Total storage space.
+
+* `resource_spec_code` - Resource specifications.
+
+## Import
+
+The rocketmq instance can be imported using the `id`, e.g.
+
+```
+$ terraform import huaweicloud_dms_rocketmq_instance.test 0ce123456a00f2591fabc00385ff1234
```

Comment:
actually, the ID format of RockMQ instance is `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.

Reply:
done